symfony/client/src/orangehrmLeavePlugin/pages/configure/holiday/SaveHoliday.vue

```diff
@@ -123,6 +125,7 @@ export default {
 
   data() {
     return {
+      yearArray: [...yearRange(201)],
```

Comment:
unless the requirement specifies, try to use the same year range due to selecting year from calendar can be hard. (50 past years <- current year -> 50 future years)

Reply:
As discussed with kavinda, there was a requirement to have date-range of 100 from both sides in the enterprise product.